### PR TITLE
PPHW 2: Substantial changes to Motivation

### PR DESCRIPTION
**Pull request by gonuke**
I propose this set of changes separately(*) because it presents a substantial reframing of this section.  It may be subject to discussion and I am look forward to such discussions if anyone feels that I've changed things too much on any axis.

There are a number of goals for these changes:

* further promote the fuel cycle analysis reasons for flexibility & extensibility as motivations
* a slightly more positive (about us) than negative (about others) tone
* stronger highlighting of the DRE
* expand open access to include development practices
* promote role of modularity in agent interchangeability as key innovation, enabled by DRE

(*) I have made this PR largely independent of others, although there are a couple of small typos that also are corrected in #80

### CODE REVIEW

File: fundamentals/fcsreqs.tex [hunk 1]
```diff
@@ -1,4 +1,52 @@
 
+\subsubsection{Open Access and Development Practices}
+
+The proprietary concerns of research institutions and security constraints of
+data within fuel cycle simulators often restrict access. Use of a simulator is
+therefore often limited to its institution of origin, necessitating effort
+duplication at other institutions and thereby squandering broader human
+resources. License agreements and institutional approval are required for most
+current simulators (i.e. \gls{COSI}6, \gls{DANESS}, \gls{DESAE}, EVOLCODE,
+FAMILY21, \gls{NFCSim})\cite{juchau_modeling_2010}, ORION, and VISION.  Even when, as in
```
Comment (katyhuff) [inline fundamentals/fcsreqs.tex]: I just noticed this instance of VISION isn't in a \gls{}, but probably should be.

File: fundamentals/fcsreqs.tex [hunk 2]
```diff
@@ -1,4 +1,52 @@
...
+the case of the MIT \gls{CAFCA} software, the source code is unrestricted, the
```
Comment (katyhuff) [inline fundamentals/fcsreqs.tex]: Maybe moving VISION alongside CAFCA would help with that concern about whether or not vision still needs institutional approval (though, I do think you have to ask Idaho for VISION still)

Comment (gonuke) [inline fundamentals/fcsreqs.tex]: Yes, that might work, although I am not sure I can easily find either of them on the interwebs?

Comment (gonuke) [inline fundamentals/fcsreqs.tex]: I can't find instructions for how to get CAFCA or VISION anywhere

Comment (katyhuff) [inline fundamentals/fcsreqs.tex]: Yeah, I think that's sufficient restriction to fall into the "institutional approval" category. Maybe we should just change the "and" to an "or".  

> License agreements "or" institutional approval

**Pull request merged**